src/javascript/Shopper/ShopNav/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react'
+
+import { NavLink, Outlet } from 'react-router-dom'
+
+const ShopNav: FC = () => {
+  return (
+    <div className="flex w-screen h-auto justify-evenly">
+      <nav className="flex">
+        <NavLink className="px-10" to="/">
+          Automodiv
+        </NavLink>
+        <div className="flex justify-self-end justify-between">
+          <NavLink className="px-10" to="/shopper">
+            Home
+          </NavLink>
+          <NavLink className="px-10" to="/shopper/join">
+            Register
+          </NavLink>
+          <NavLink className="px-10" to="/shopper/login">
+            Login
+          </NavLink>
+        </div>
+      </nav>
+      <main>
+        <Outlet />
```

Comment:
This outlet acts just like `children` in react, so its like a wrapper. So what ever you have in between `<Route {...params}>{Elements In Here}</Route>` will show.

Reply:
Copy pasta from the other routes, did not drill down and clean up.